tests/test-cases/readers/cube-config/v1.4/multi-measure-pivoted-dataset-units-and-attributes.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
```

Comment:
Rob says he wants it at `v1` since we're actually testing the `describes_observations` functionality here - and using `v1` instead of specifying a minor version will mean that the test automatically ensures the functionality remains in all future minor versions of the the v1.x schema.